OrbitCore/OrbitModule.cpp

```diff
@@ -108,45 +107,62 @@ Function* Pdb::FunctionFromName(const std::string& a_Name) {
 }
 
 //-----------------------------------------------------------------------------
-std::string FindSymbols(const std::string& module_path) {
+std::unique_ptr<ElfFile> FindSymbols(std::string& module_path) {
+  auto module_elf_file = ElfFile::Create(module_path);
+  if (module_elf_file == nullptr) {
+    PRINT(absl::StrFormat("Unable to load module (object file) from \"%s\"",
+                          module_path));
+    return nullptr;
+  }
+
+  if (module_elf_file->ContainsSymbols()) {
+    return module_elf_file;
+  }
+
   // Look for .debug files associated with passed in module name
-  // TODO: .debug file might not have same name as module, we should check
-  //       for unique identifier in the symbols file...
+  // TODO: .debug file might not have same name as module, maybe there is a good
+  //       way of searching files based on buildId
 
   std::string dir = Path::GetDirectory(module_path);
-  std::vector<std::string> symbolDirectories = {"~/", Path::GetHome(), dir,
-                                                dir + "debug_symbols/", "/home/cloudcast/"};
-
   std::string file = Path::StripExtension(Path::GetFileName(module_path));
 
-  for (auto& symbolDirectory : symbolDirectories) {
-    std::string debugFile = symbolDirectory + file + ".debug";
-    if (Path::FileExists(debugFile)) {
-      PRINT_VAR(debugFile);
-      return debugFile;
+  std::vector<std::string> searchDirectories = {
```

Comment:
I think we should use snake_case for local variables. Or at least be consistent with the other variables in the same function.